roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -123,3 +123,56 @@
     volume_mounts: "{{ openshift_hosted_registry_volumes }}"
     edits: "{{ openshift_hosted_registry_edits }}"
     force: "{{ True|bool in openshift_hosted_registry_force }}"
+
+- block:
```

Comment:
I thought about that too. If I did, I don't think I'd need a block statement, yeah?